R/CohortDefinition.R

```diff
@@ -634,3 +639,54 @@ deleteCohortDefinition <- function(cohortId, baseUrl, silent = FALSE, stopOnErro
   return(NA)
 }
 
+
+#' Get cohort generation report
+#'
+#' @details
+#' Obtains a list with dataframe containing details of report for cohort generation output.
+#' 
+#' @param baseUrl     The base URL for the WebApi instance, for example: "http://server.org:80/WebAPI".
+#' @param cohortId    The Atlas cohort definition id for the cohort
+#' @param sourceKey   The source key for a CDM instance in WebAPI, as defined in the Configuration page
+#' @param mode        Mode is used to differentiate between inclusion rules and count by events (mode = 0, default) 
+#'                    or persons (mode = 1). Default value = 0. 
+#' @return            A list of data frames containing cohort generation report
+#'
+#' @export
+getCohortGenerationReport <- function(baseUrl, cohortId, sourceKey, mode = 0) {
```

Comment:
This seems very similar to `getCohortGenerationStatuses`. Perhaps a more informative name? (I can't think of one, so maybe not)

Reply:
This function is returning the output of cohort generation process - so maybe `getOutputFromCohortGeneration` ?

but it wont follow the pattern in precedence `getCohortGenerationStatuses`

how about `getCohortGenerationOutput` - but it is not very far from `getCohortGenerationReport`. Output is a predecessor for Report - so maybe Output is a better choice vs report.